loaders/authentication.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
+
+//base URL of cigarra authentication
+const URL = 'https://sigarra.up.pt/feup/pt/mob_val_geral.autentica?'
+
+//build URL with credentials
+let urlRequest = URL + 'pv_login=' + process.env.UP_USER + '&pv_password=' + process.env.UP_PWD;
+
+let cookie = null;
+
+//Called sigarra api to make authentication
+async function getCookie(){
+   try {
+        const response = await fetch(urlRequest, {
+            method: 'POST'
+        });
+        if(response.ok){
+            const res = await response.json();
+            return res;
+        }
+        throw new Error('Request Failed!');
+
+   }catch (error){
+       console.log(error);
+   }
+}
+
+//give value to cookie
+async function returnCookie(){
```

Comment:
I would call this method 'getCookie()' and the one above 'updateCookie()' or 'refreshCookie()'

But I leave that decision to you

Reply:
Yeah it is better